### ISSUE TITLE
[Bug]: After updating its like a normal Youtube music app, cant even press X to put on tray, and dont have options to do anything

### ISSUE THREAD

**Issue by Dremali** [th-ch/youtube-music#2113]
### Preflight Checklist

- [X] I use the latest version of YouTube Music (Application).
- [X] I have searched the [issue tracker](https://github.com/th-ch/youtube-music/issues) for a bug report that matches the one I want to file, without success.
- [X] I understand that **th-ch/youtube-music has NO affiliation with Google or YouTube**

### YouTube Music (Application) Version

3.3.9 - 3.3.10

### Last Known Working YouTube Music (Application) version

3.3.8

I cant get to use the plugins after the updated I cant even see them

### Additional Information
Fixed the problem by following @Xydem4 Instructions, Unistalling the app and then deleting the %AppData%

**Comment by Xydem4** [comment]
sounds like what a few of us were experiencing after trying to update to anything past 3.3.6. Had to completely uninstall the app and remove all the files from appdata/roaming/Youtube Music. Then reinstalled and it's working fine. just had to relog in and reset everything how i had it before.

https://github.com/th-ch/youtube-music/issues/2105#issuecomment-2143809383

**Comment by ov1d1u** [comment]
This worked for me, too. I used AppCleaner (on MacOS) to completely wipe the app then I reinstalled again.

**Comment by sofijacom** [comment]
I don’t know what you’re writing about, but application v3.3.10 works great for me
![screenshot-20240603-160111](https://github.com/th-ch/youtube-music/assets/107557749/83d92ad5-2aa6-4037-bea9-64d443e8c024)

**Comment by ShadowBullet25** [comment]
I was having this exact same issue. this happened on the last 2 versions. after uninstalling from windows apps page, and deleting appdata and reinstalling from scratch, the app was back to normal. i then dragged my previous config.json from the appdata folder back in to import my previous settings, and then the tabs at the top went back to normal youtube music instead of how the client was supposed to be like. 

i tested, and deleting config.json after signing in kept me signed in, but the app was back to normal. however you need to put your preferences for the app back to normal afterwards. 

i dont know if this will help the developer (as it could be a plugin we all had enabled that was in common that was causing this), but here is my config.json for youtube music that was working before the last 2 versions: 

[config.json](https://github.com/user-attachments/files/15536052/config.json)

**Comment by Xydem4** [comment]
It's possible that it may be something in the config file. But I doubt it's enabled plugin related. I tried a few things while i was waiting for a fix. One of them being to disable every plugin and make as many settings possible stock before installing the new version. was the same result. i believe we had 3.3.9 when i was trying that.

**Comment by ShadowBullet25** [comment]
> It's possible that it may be something in the config file. But I doubt it's enabled plugin related. I tried a few things while i was waiting for a fix. One of them being to disable every plugin and make as many settings possible stock before installing the new version. was the same result. i believe we had 3.3.9 when i was trying that.

i looked in the config files, and the formats seem to be fairly different though. when you update the application, is it also supposed to update the config to reflect the new format?

**Comment by anthropos9** [comment]
When I deleted my `~/.config/YouTube\ Music` directory everything worked correctly. Of course that meant having to re-login and update all the settings again, but it worked. Which seems to lend credibility to the idea there's something up in the config files causing the bugs.

**Comment by ralphbedayo** [comment]
> When I deleted my `~/.config/YouTube\ Music` directory everything worked correctly. Of course that meant having to re-login and update all the settings again, but it worked. Which seems to lend credibility to the idea there's something up in the config files causing the bugs.

This helped fix mine in Linux Mint. Might be applicable for other distros too.

**Comment by GardenOfWyers** [comment]
I found the issue for my instance which seemed to show similar symptoms as yours - no "Plugins", "Options", "Navigation" in the menu bar. 
![image](https://github.com/th-ch/youtube-music/assets/43160823/3a9613ec-f3b5-415b-ba4d-3268f8f9684a)
As other's pointed out, it seems to be an issue with the `config.json` file found in `%appdata%\YouTube Music` where my instance of `config.json` was carried over from many version ago, so it may not have been updated to include any required keys in it. Anyway, the key that I added to my migrated `config.json` that seemed to fix it was `"themes": []` in the `"options"` section. I added that and then closed and re-opened the app, and the menu was back with all my settings intact. 
![Screenshot 2024-06-06 122056](https://github.com/th-ch/youtube-music/assets/43160823/d9fd9053-65d8-4a9c-9edd-6cf284e1762f)
Hopefully this helps.
